lib/supavisor/db_handler.ex

```diff
@@ -31,6 +31,12 @@ defmodule Supavisor.DbHandler do
   @spec cast(pid(), pid(), binary()) :: :ok | {:error, any()} | {:buffering, non_neg_integer()}
   def cast(pid, caller, msg), do: :gen_statem.cast(pid, {:db_cast, caller, msg})
 
+  @spec set_idle(pid()) :: :ok
+  def set_idle(pid), do: :gen_statem.cast(pid, :set_idle)
+
+  @spec change_skt_owner(pid(), pid()) :: {:ok, S.sock()}
+  def change_skt_owner(pid, caller), do: :gen_statem.call(pid, {:tcp_owner, caller}, 15_000)
```

Comment:
```suggestion
  @spec change_socket_owner(pid(), pid()) :: {:ok, S.sock()}
  def change_socket_owner(pid, caller), do: :gen_statem.call(pid, {:tcp_owner, caller}, 15_000)
```

Do not shorten name as it make it less readable.

Reply:
If you want to shorten it, then `sock` would be IMHO better than `skt`, and it is only 1 character longer.